apps/console/src/extensions/components/subscription/api/subscription.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import useRequest, { 
+    RequestConfigInterface,
+    RequestErrorInterface,
+    RequestResultInterface 
+} from "apps/console/src/features/core/hooks/use-request";
+import { HttpMethods } from "modules/core/src/models";
+import { getDomainQueryParam } from "../../tenants/api/tenants";
+import { getTenantResourceEndpoints } from "../../tenants/configs";
+import { TenantTierRequestResponse } from "../models/subscription";
+
+export const useTenantTier = <Data = TenantTierRequestResponse, 
```

Comment:
Lets add some Doc comments.
Refer the `oranizationDiscovery/api` feature.
```suggestion
export const useGetTenantTier = <Data = TenantTierRequestResponse, 
```

Reply:
Any specific reason to use default exports? AFAIK, default exports are usually discouraged.